CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.2] - 2018-11-08
+### Added
+* Author email and improve description
+* Data section of demos readme
+* Dash, html and renderer to install requirements
```

Comment:
Let's be explicit here. "Added the components "dash", "dash-html-components", and "dash-renderer" as explicit package requirements."

Reply:
Updated!